packages/framework/presence/src/presence.ts

```diff
@@ -0,0 +1,127 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import type { ConnectedClientId } from "./baseTypes.js";
+import type { NotificationsManager } from "./notificationsManager.js";
+import type {
+	PresenceStates,
+	PresenceWorkspaceAddress,
+	PresenceStatesSchema,
+} from "./types.js";
+
+import type { ISubscribable } from "@fluid-experimental/presence/internal/events";
+import type { InternalTypes } from "@fluid-experimental/presence/internal/exposedInternalTypes";
+
+/**
+ * A client within a Fluid session (period of container connectivity to service).
+ *
+ * @sealed
+ * @alpha
+ */
+export interface ISessionClient {
+	/**
+	 * Get current client connection id.
+	 *
+	 * @returns Current client connection id.
+	 *
+	 * @remarks
+	 * Connection id will change on reconnection.
+	 */
+	currentClientId(): ConnectedClientId;
```

Comment:
If this is the only member, do we need ISessionClient?  Can those usages instead just be a ConnectedClientId directly?